packages/backend/src/studio-api-impl.ts

```diff
@@ -240,7 +241,26 @@ export class StudioApiImpl implements StudioAPI {
 
     if (!connection) throw new Error('no running container provider connection found.');
 
-    return this.applicationManager.requestPullApplication(connection, recipe, model);
+    let model: ModelInfo | undefined;
+    if (isRecipePullOptionsWithModelInference(options)) {
```

Comment:
why do we need `isRecipePullOptionsWithLlamaStack` and `isRecipePullOptionsWithModelInference` when you introduced the `startLlamaStack` boolean ?

> isRecipePullOptionsWithLlamaStack seems to have no usage?